css/footer.css

```diff
@@ -16,3 +16,213 @@
+  width: 327px;
+  height: 48px;
+  border-radius: 4px;
+  border: 1px solid #d0d9d4;
+  background: #fff;
+}
+
+.femail {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin-top: 24px;
+  color: #6b778c;
+  font-style: normal;
+  padding: 15px;
+}
+
+form textarea {
+  width: 327px;
+  height: 114px;
+  border-radius: 4px;
+  border: 1px solid #60c095;
+  margin-top: 24px;
+  background: #fff;
+}
+
+.ftextarea {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin-top: 24px;
+  color: #091e42;
+  font-style: normal;
+  padding-left: 16px;
+  padding-top: 12px;
+}
+
+.ftextarea::placeholder {
+  color: black;
+}
+
+.get-in-touch {
+  padding: 12px;
+  background: #36b37f;
+  font-family: "Inter", sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  border-style: none;
+  border-radius: 4px;
+  margin-top: 24px;
+  margin-left: 99px;
+}
+
+.footer-social-icons {
+  display: flex;
+  justify-content: center;
+  gap: 20px;
+  align-items: center;
+  padding: 24px;
+}
+
+.footer-social-icons-container {
+  position: relative;
+  display: flex;
+  justify-content: center;
+}
+
+.footer-frame-mob {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+
+.footer-email {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #42526e;
+  font-style: normal;
+  text-align: center;
+  padding-top: 19px;
+}
+
+.footer-hr {
+  display: flex;
+  justify-content: center;
+}
+
+hr {
+  margin: 100px 120px 10px 121px;
+  width: 134px;
+  height: 4px;
+  background: #0d151e;
+  border-radius: 10px;
+}
+
+@media (max-width: 768px) {
+  .footer-img {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .footer-frame-mob,
+  .footer-email,
+  .footer-hr {
+    display: none;
+  }
+
+  .footer-desktop-container {
+    display: grid;
+    grid-template-columns: 463px auto;
+    gap: 24px;
+    padding-top: 80px;
+    padding-bottom: 154px;
+    margin: 0 134px;
+    position: relative;
+  }
+
+  .last-msg {
+    margin: 0;
+  }
+
+  .last-msg-text {
+    color: #172b4d;
+    font-family: "Inter", sans-serif;
+    font-weight: 700;
+    font-size: 30px;
+    text-align: center;
+    line-height: 52px;
+    font-style: normal;
+    padding-top: 50px;
+  }
+
+  .contact-form {
+    width: 100%;
+  }
+
+  form input {
+    width: 584px;
+    height: 48px;
+  }
+
+  form textarea {
+    width: 584px;
+    height: 114px;
+  }
+
+  .get-in-touch {
+    margin: 0;
+    margin-top: 24px;
+  }
+
+  .footer-img {
+    padding-bottom: 90px;
+    border-bottom: 2px solid #dfe1e6;
+  }
+
+  .footer-img1 {
+    position: absolute;
+    top: 3320px;
+  }
+
+  .footer-img3 {
+    position: absolute;
+    left: 1040px;
+    top: 3200px;
+  }
+
+  .footer-img2 {
+    position: absolute;
+    left: 274px;
+    bottom: -3000px;
+  }
```

Comment:
- [ ] Kindly check and ensure that the image close to the right side of the form is not too close to the textarea field in order to match the actual design. 
- [ ] The social media icons are out of position compared to the original design. 
Kindly check and fix this issures to make your site more professional 

### Your design 


![Screenshot from 2023-02-25 12-05-25](https://user-images.githubusercontent.com/65233860/221356807-e23d8c3b-8380-47cc-bfb2-7606c69c5dd5.png)


### Actual design 


![Screenshot from 2023-02-25 12-07-19](https://user-images.githubusercontent.com/65233860/221356815-06e942c2-3b22-409e-8b8b-27e4fd61233e.png)